full-stack-tests/backend/src/integration/CloudSqlite.test.ts

```diff
@@ -188,6 +191,90 @@ describe("CloudSqlite", () => {
     container.disconnect({detach: true});
   });
 
+  it("Should a single user upload changes successfully even if write lock expires", async () => {
+    const testContainer1 = testContainers[0];
+    const testCache = azSqlite.makeCaches(["testCache"])[0];
+    testContainer1.connect(testCache);
+    testContainer1.acquireWriteLock(user1);
+    await testContainer1.copyDatabase("testBim", "testBimCopy");
+    const db = await BriefcaseDb.open({ fileName: "testBimCopy", container: testContainer1 });
+    db.saveFileProperty({ name: "logMask", namespace: "logMaskTest", id: 1, subId: 1 }, "this is a test");
+    db.close();
+    // set the expires time to five mins earlier, which avoids waiting for 5 mins here.
+    await azSqlite.subtractFromCurrentWriteLockExpiryTime(testContainer1, "bcv_kv.bcv", 5);
+    await expect(testContainer1.uploadChanges()).to.eventually.be.fulfilled;
```

Comment:
Is `eventually.be.fulfilled` enough to verify a successfully upload?

Reply:
The point of `await expect(testContainer1.uploadChanges()).to.eventually.be.fulfilled;` is to check that no error is thrown from `RequireWriteLock()` on the native side. We don't really care if the upload itself is successful or not imo.

Reply:
Makes sense, just wanted to double check